Makefile

```diff
@@ -19,6 +19,7 @@ setup:
 	@pip list | grep urllib3 >/dev/null && pip uninstall -y urllib3 || true
 	@pip -q uninstall -y setuptools || true
 	@yum -q reinstall -y python-setuptools
+	@pip install lxml
```

Comment:
why not add python-lxml to yum? or this package is not present in rhel repos?

Reply:
we can do it with yum, sure. I'll do it in the dockerfile